src/dashboard/home-dashboard.component.tsx

```diff
@@ -2,22 +2,94 @@ import React from "react";
 import styles from "./home-dashboard.component.css";
 import { Link, match } from "react-router-dom";
 import { UserHasAccessReact } from "@openmrs/esm-api";
+import { useConfig } from "@openmrs/esm-module-config";
+import DashboardButton from "../dashboard-button/dashboard-button.component";
+
+const buttonDefs = [
```

Comment:
These are the buttons that are present on the demo RefApp. These will of course be made configurable.

Reply:
Can you post picture?

Reply:
![Screenshot from 2020-02-12 18-14-59](https://user-images.githubusercontent.com/1031876/74389407-b2167880-4dc3-11ea-8854-348cd65c1558.png)
![Screenshot from 2020-02-12 18-15-14](https://user-images.githubusercontent.com/1031876/74389411-b347a580-4dc3-11ea-91f5-1b5781cb123f.png)

This disagrees in terms of color with the designs -- I'd want to add the designed style to the styleguide. And I'd want to call that a separate PR.